File: Lenguajes Programación I/JS/Tipos de Datos y Sintaxis/Ejercicios Lógicos/areacirculo.js
function calcularAreaCirculo(radio) {
    let area = Math.PI * Math.pow(radio, 2);
    return area;
}
   // Asignación de la variable radio que equivale a 5
   let radio = 5;
   let areaCirculo = calcularAreaCirculo(radio);
   //Imprimir el resultado en la consola
   console.log("El área del circulo de radio " + radio + " es: " + areaCirculo);
   

File: Conceptos Básicos de Lógica de Programación/Python/Funciones y Procedimientos I/areaCirculo.py
def area_circulo(radio):
    valordePi = 3.1416
    area = valordePi * (radio ** 2)
    return area

def ejercicio_funciones_2():
    radio = float(input("Escribe el radio: "))
    area = area_circulo(radio)
    print(area)

ejercicio_funciones_2()
